src/main/proto/com/google/api/codegen/samplegen/v1/sample_config_v1.proto

```diff
@@ -0,0 +1,246 @@
+syntax = "proto3";
+
+package com.google.api.codegen.samplegen.v1;
```

Comment:
I'm not completely sure about the `.v1` here, because we're already specifying the version in `schema_version`-- the repetition troubles me slightly. I guess it's alright if we have the major  version number as part of the package, and we then require `schema_version` to be consistent with this major version and to further specify the minor version.

Reply:
I must confess that I don't understand proto package names well enough to assert appending `v1` here is correct (hopefully will gain more knowledge during implementing this config), but my understanding is:
The `schema_version` field should always be there regardless of the version of sample configuration. When the generator loads the config yaml specified by users, it will check the `schema_version` field first, and then choose the correct version of config proto based on the value of this field to convert the yaml into -- and so that the generator needs to store all possible versions (i.e., supported and not deprecated) of schemas written in protos here, hence the package `v1` here. And I think it should be like:

```
com.google.api.codegen.samplegen.v1.sample_config_v1.proto 
 ^^^^^ load this when the yaml has schema_version field value set to 1.0.0,

com.google.api.codegen.samplegen.v1p2.sample_config_v1p2.proto
^^^^^ load this when the yaml has schema_version field value set to 1.2.0
```

Also note that the name of proto file is largely ignored, so package name is probably _the_ identifier here.